docs/pages/documentation/configuration-file.md

```diff
@@ -58,6 +58,104 @@ This section documents utilization of the configuration file in place of CLI opt
       * Creates a test case for each action executed (reruns included)
   * Example: `jUnitReportSpecificity: testReport`
 
+### reportState
```

Comment:
Looks good, however, I noticed that in the code you are using `reportStates` with an `s` and in this documentation file it is referred to as `reportState`.